pages/04.tutorials/27.setup-an-nginx-ingress-controller-with-the-proxy-protocol/default.en.md

```diff
@@ -112,6 +112,23 @@ The example below configures the nginx ingress controller after the first update
     type: LoadBalancer
 ```
 
+## Additional configuration options. Use a custom floating ip network (optional)
+
+The example below configures the nginx ingress controller below to use a custom floating ip network. You will need to add the floating ip network id you have created to the service.
+By default the standard metakube floating ip network will be used and no further configuration is required.
+
+### Additional yaml to add to the nginx ingress controller for a custom floating ip network (optional)
+
+```yaml
+  service:
+    enabled: true
+
+    annotations:
+      loadbalancer.openstack.org/floating-network-id: "*****"
```

Comment:
Is this something only for octavia or can this be put into the general doc like https://docs.syseleven.de/metakube/en/tutorials/create-a-load-balancer#keeping-load-balancer-floating-ips-on-openstack?

Reply:
Ah good catch. This is a general annotation that can be used in general. 👍

Reply:
then it should suffice in the general doc and this can be removed here :)

Reply:
Linked to the generic page now instead :)